app/views/admins/users/edit.html.slim

```diff
@@ -0,0 +1,19 @@
+.container
+  h2.text-center = t '.edit_user_info'
+
+  = form_for(@user, url: admins_user_path(@user), remote: false) do |f|
+    .alert.alert-danger
+      = render "devise/shared/error_messages", resource: @user 
```

Comment:
display errors under appropriate field... so under `email` input field should be error related to email